src/Controller/EventsController.php

```diff
@@ -563,14 +563,14 @@ public function putAction(Request $request, PDO $db): void
             }
 
             $event['cfp_start_date'] = null;
-            $cfp_start_date          = $request->getParameter('cfp_start_date', false);
+            $cfp_start_date          = $request->getStringParameter('cfp_start_date', '');
 
             if (false !== $cfp_start_date && strtotime($cfp_start_date)) {
```

Comment:
If we're defaulting to '' then I think we need !== '' here

Reply:
Fixed